src/hb-ot-layout.cc

```diff
@@ -279,10 +279,9 @@ hb_ot_layout_get_ligature_carets (hb_font_t      *font,
 				  hb_codepoint_t  glyph,
 				  unsigned int    start_offset,
 				  unsigned int   *caret_count /* IN/OUT */,
-				  int            *caret_array /* OUT */)
+				  hb_position_t  *caret_array /* OUT */)
```

Comment:
Now it matches with its actual signature on [src/hb-ot-layout.h](https://github.com/ebraminio/harfbuzz/blob/master/src/hb-ot-layout.h#L88)